src/handlers/assign.rs

```diff
@@ -487,6 +487,15 @@ pub(super) async fn handle_command(
                     name.to_string()
                 } else {
                     let teams = crate::team_data::teams(&ctx.github).await?;
+                    // Determine if assignee is a team. If yes, add the corresponding label
+                    // team name here is without prefix 't-' (e.g. 'compiler', 'libs', etc.)
+                    if let Some(team) = teams.teams.get(&name) {
+                        let _team_label = format!("t-{}", team.clone().name);
+                        issue
+                            .add_labels(&ctx.github, vec![github::Label { name: _team_label }])
```

Comment:
I think this will fail if the repository does not have a `T-*` label for the given team. Would it be possible to ignore the `UnknownLabels` error (or just log it)?

(Also, not all teams use T-* labels that match their name exactly.)

Stylistically, is there a reason to use a leading underscore?

Reply:
thanks @ehuss for giving a first look!

Yes, this solution is only useful for the main teams (compiler, libs-*, rustdoc). AFAICS, there's no way to infer the github label from the team name, it's not exposed in the [teams.json](https://team-api.infra.rust-lang.org/v1/teams.json) file. Any idea to improve the chances to catch a team label that does not respect the `T-<teamname>` structure (besides fixed mappings).

> Stylistically, is there a reason to use a leading underscore?

old habits? :sweat_smile: changed it (in case you meant it was ugly).

Reply:
> (Also, not all teams use T-* labels that match their name exactly.)

Well, now that I think of it. It wouldn't be a bad idea to map teams+labels in the team.json by retrieving this info from the .toml file. Ex. [the compiler team](https://github.com/rust-lang/team/blob/71719e35d2dd03b1741f2cf8af22f1db707524ce/teams/compiler.toml#L43C12-L43C12) has it. Sort of.

Anyway nothing I'd think about here and now.

Reply:
tentatively marking the review as resolved with the latest commit